### PR TITLE
.Net: Added README to FunctionInvocationApproval demo app

### PR DESCRIPTION
**Pull request by dmytrostruk**
### Motivation and Context

Fixes: https://github.com/microsoft/semantic-kernel/issues/6739

### CODE REVIEW

**Review by SergeyMenshykh — APPROVED** [review]

File: dotnet/samples/Demos/FunctionInvocationApproval/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Function Invocation Approval
+
+This console application shows how to use function invocation filter (`IFunctionInvocationFilter`) to invoke Kernel Function only if such operation was approved.
```
Comment (markwallace-microsoft) [inline dotnet/samples/Demos/FunctionInvocationApproval/README.md:3]: `to invoke a Kernel Function`

File: dotnet/samples/Demos/FunctionInvocationApproval/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+If function invocation was rejected, the result will contain an information about this, so LLM can react accordingly.
+
+Application uses a sample plugin that allows to build a software by following development stages: collection of requirements, design, implementation, testing and deployment.
+
+Each step can be approved or rejected. Based on that, LLM will decide how to proceed.
```
Comment (markwallace-microsoft) [inline dotnet/samples/Demos/FunctionInvocationApproval/README.md:8]: `Based on that, the LLM will`

File: dotnet/samples/Demos/FunctionInvocationApproval/README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Function Invocation Approval
+
+This console application shows how to use function invocation filter (`IFunctionInvocationFilter`) to invoke Kernel Function only if such operation was approved.
+If function invocation was rejected, the result will contain an information about this, so LLM can react accordingly.
+
+Application uses a sample plugin that allows to build a software by following development stages: collection of requirements, design, implementation, testing and deployment.
```
Comment (markwallace-microsoft) [inline dotnet/samples/Demos/FunctionInvocationApproval/README.md:6]: `The application uses a sample plugin which builds software by following these development stages: `

File: dotnet/samples/Demos/FunctionInvocationApproval/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+# Function Invocation Approval
+
+This console application shows how to use function invocation filter (`IFunctionInvocationFilter`) to invoke Kernel Function only if such operation was approved.
+If function invocation was rejected, the result will contain an information about this, so LLM can react accordingly.
```
Comment (markwallace-microsoft) [inline dotnet/samples/Demos/FunctionInvocationApproval/README.md:4]: `will contain the reason why, so the LLM can respond appropriately.`

**Review by markwallace-microsoft — APPROVED** [review]

**Review by SergeyMenshykh — APPROVED** [review]

**Pull request merged**